Resources/Audio/Machines/Arcade/attributions.yml

```diff
@@ -0,0 +1,4 @@
+- files: ["hahaha.ogg", "pacman_eat.ogg", "pacman.ogg", "pew_pew.ogg", "sting_*.ogg"]
```

Comment:
These sound like they're spedup samples from pacman so I don't think these are useable for licensing reasons.

Reply:
I made all of these from scratch on a Korg Kaossilator KO-1.

Reply:
But let me know if the Pacman ones are too blatantly similar and I'll replace them.

Reply:
I'll just take those two out.